bedrock/firefox/tests/test_redirects.py

```diff
@@ -100,319 +100,350 @@ def test_mobile_app():
             False,
         ),
         (
-            "/firefox/ios/testflight/",
-            f"https://www.firefox.com/channel/ios/testflight/{EXPECTED_REDIRECT_QS}",
+            "/en-US/firefox/ios/testflight/",
+            f"https://www.firefox.com/en-US/channel/ios/testflight/{EXPECTED_REDIRECT_QS}",
             EXPECTED_FIREFOX_COM_REDIRECT_CODE,
             False,
         ),
         (
-            "/firefox/linux/",
-            f"https://www.firefox.com/browsers/desktop/linux/{EXPECTED_REDIRECT_QS}",
+            "/en-US/firefox/linux/",
+            f"https://www.firefox.com/en-US/browsers/desktop/linux/{EXPECTED_REDIRECT_QS}",
             EXPECTED_FIREFOX_COM_REDIRECT_CODE,
             False,
         ),
         (
-            "/firefox/mac/",
-            f"https://www.firefox.com/browsers/desktop/mac/{EXPECTED_REDIRECT_QS}",
+            "/en-US/firefox/mac/",
+            f"https://www.firefox.com/en-US/browsers/desktop/mac/{EXPECTED_REDIRECT_QS}",
             EXPECTED_FIREFOX_COM_REDIRECT_CODE,
             False,
         ),
         (
-            "/firefox/more/",
-            f"https://www.firefox.com/more/{EXPECTED_REDIRECT_QS}",
+            "/en-US/firefox/more/",
+            f"https://www.firefox.com/en-US/more/{EXPECTED_REDIRECT_QS}",
             EXPECTED_FIREFOX_COM_REDIRECT_CODE,
             False,
         ),
         (
-            "/firefox/set-as-default/",
-            f"https://www.firefox.com/landing/set-as-default/{EXPECTED_REDIRECT_QS}",
+            "/en-US/firefox/set-as-default/",
+            f"https://www.firefox.com/en-US/landing/set-as-default/{EXPECTED_REDIRECT_QS}",
+            EXPECTED_FIREFOX_COM_REDIRECT_CODE,
+            False,
+        ),
+        (
+            "/en-US/firefox/set-as-default/thanks/",
+            f"https://www.firefox.com/en-US/landing/set-as-default/thanks/{EXPECTED_REDIRECT_QS}",
             EXPECTED_FIREFOX_COM_REDIRECT_CODE,
             False,
         ),
         (
-            "/firefox/set-as-default/thanks/",
-            f"https://www.firefox.com/landing/set-as-default/thanks/{EXPECTED_REDIRECT_QS}",
+            "/en-US/firefox/unsupported-systems/",
+            f"https://www.firefox.com/en-US/browsers/unsupported-systems/{EXPECTED_REDIRECT_QS}",
             EXPECTED_FIREFOX_COM_REDIRECT_CODE,
             False,
         ),
         (
-            "/firefox/unsupported-systems/",
-            f"https://www.firefox.com/browsers/unsupported-systems/{EXPECTED_REDIRECT_QS}",
+            "/en-US/firefox/windows/",
+            f"https://www.firefox.com/en-US/browsers/desktop/windows/{EXPECTED_REDIRECT_QS}",
             EXPECTED_FIREFOX_COM_REDIRECT_CODE,
             False,
         ),
         (
-            "/firefox/windows/",
-            f"https://www.firefox.com/browsers/desktop/windows/{EXPECTED_REDIRECT_QS}",
+            "/de/firefox/windows/",
+            f"https://www.firefox.com/de/browsers/desktop/windows/{EXPECTED_REDIRECT_QS}",
```

Comment:
Are there any tests that are checking other query params are being preserved as expected? I didn't see any in this file there's plenty of places they could be.

Reply:
I've gone with the idea that if some query params eg the ones for `installer-help/` are preserved, all of them will be. If you can suggest some for more exhaustive testing, we can add them, np

Reply:
eg see test_springfield_redirects_carry_over_querystrings_and_add_redirect_source